docs/guides/operator-guide-aws-marketplace.md

```diff
@@ -239,6 +239,15 @@ To check the UID2 Operator status of your Load Balancer, complete the following
 1. Identify the DNS name of your load balancer by going to **EC2 > Load balancers** and looking at the **DNS name** column of your load balancer.
 2. In your browser, go to `https://{dns-name-of-your-load-balancer}/ops/healthcheck`. A response of `OK` indicates good operator status.
 
+## Private Operator Attestation Failure
```

Comment:
Here's my suggestion, let me know if I have anything wrong on this:

## Private Operator Attestation Failure

If the Private Operator fails to attest with the UID2 Core service, it fails. This might happen in either of the following cases:

- HTTP 401 response. If your operator key receives a 401 response from the UID2 Core service, the Private Operator terminates itself immediately.
- Any other non-200 response code. If your attest request receives any response code other than a 200 or a 401, the Private Operator continues to function for 12 hours. If the issue is not resolved in this time frame, the Private Operator terminates itself.

Private Operator hosts must have infrastructure in place to manage alerting and and to restart the Operator in the case of an error.